src/parser.ts

```diff
@@ -158,41 +160,54 @@ export function parseLogStream(logStream: QuasiSubject<string>): QuasiObservable
           meta,
           rowIndex: progress.index++,
           row,
+          info: parseJsonInfo(row.Info),
         })
-      });
+      }).catch(reason => console.error(reason));
```

Comment:
We should propagate this error back out instead of swallowing it.

Reply:
Yep, good catch. This was something I was using while testing/debugging.